src/components/TrackMetadataPopper.tsx

```diff
@@ -0,0 +1,321 @@
+              {newAttrs[key].value || oldAttrs[key].value}
+            </AttributeValue>
+          </Attribute>
+        ))}
+      </ChangesList>
+    </PopperContainer>
+  )
+}
+
+export const TrackMetaButton = styled.div`
+  display: none;
+  width: 27px;
+  height: 27px;
+  border-radius: 50%;
+  border: solid 1px #e2e2e2;
+  align-items: center;
+  justify-content: space-around;
+  margin: 2px;
+  cursor: pointer;
+
+  background: ${(props) => props.theme.colors.background.primary};
+
+  :hover {
+    background: ${(props) => props.theme.colors.background.selected};
+
+    path {
+      stroke: ${(props) => props.theme.colors.brand.medium};
+    }
+  }
+`
+
+export const TrackChangesButton = styled(TextButton)`
+  font-size: ${(props) => props.theme.font.size.normal};
+  color: ${(props) => props.theme.colors.text.tertiary};
+  text-decoration: underline;
+  margin-right: ${(props) => props.theme.grid.unit * 2}px;
+
+  &:hover,
+  &:focus {
+    color: ${(props) => props.theme.colors.text.tertiary} !important;
+  }
+`
+
+const RejectButton = styled(TrackChangesButton)`
+  color: ${(props) => props.theme.colors.text.secondary};
+`
+
+const PopperContainer = styled.div`
+  border-radius: ${(props) => props.theme.grid.radius.default};
+  background: ${(props) => props.theme.colors.background.primary};
+  box-shadow: 0 4px 9px 0 rgba(0, 0, 0, 0.3);
+  z-index: 10;
+`
+
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px;
+`
+
+const Label = styled.div`
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props) => props.theme.font.size.medium};
+  font-weight: ${(props) => props.theme.font.weight.normal};
+  line-height: ${(props) => props.theme.font.lineHeight.large};
+  font-style: normal;
+  padding: 2px;
+`
+
+const LabelContainer = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 4px;
+`
+
+const Attribute = styled.div`
+  padding: ${(props) => props.theme.grid.unit * 3}px;
+  cursor: pointer;
+
+  &:hover {
+    background: ${(props) => props.theme.colors.background.fifth};
+  }
+`
+
+const AttributeLabel = styled.div`
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props) => props.theme.font.size.small};
+  font-weight: ${(props) => props.theme.font.weight.bold};
+  font-style: normal;
+  line-height: 16px;
+`
+
+const AttributeValue = styled(AttributeLabel)`
+  font-weight: ${(props) => props.theme.font.weight.normal};
+`
+
+const OldAttribute = styled(AttributeValue)`
+  text-decoration: line-through;
+`
+
+const ChangesList = styled.div`
+  padding: ${(props) => props.theme.grid.unit * 2}px
+    ${(props) => props.theme.grid.unit * 4}px;
+  max-height: ${(props) => props.theme.grid.unit * 124}px;
+  background: ${(props) => props.theme.colors.background.secondary};
+  overflow-y: scroll;
+  overflow-x: hidden;
+
+  ::-webkit-scrollbar {
+    width: 12px;
+  }
+
+  ::-webkit-scrollbar-track {
```

Comment:
I would refrain from styling scrollbars both not to solve potential appearance discrepancies in different browsers on different OS and as a general accessibility practice. Also, I don't think we style scrollbars in the app in other places.

Reply:
removed